linter.py

```diff
@@ -19,12 +19,6 @@ class Xmllint(Linter):
 
     syntax = 'xml'
     cmd = 'xmllint --noout * -'
-    regex = (
-        r'^.+?:'
-        r'(?P<line>\d+):.+?: '
-        r'(?P<message>[^\r\n]+)\r?\n'
-        r'[^\r\n]*\r?\n'
-        r'(?P<col>[^\^]*)\^'
-    )
+    regex = r'^-:(?P<line>\d+):.+?: (?P<message>[^\r\n]+)(\r?\n[^\r\n]*\r?\n(?P<col>[^\^]*)\^)?'
```

Comment:
I think, looking at the example output, you need to account for possible white space between `^` and `-:`. So, start with `^\s*-:`.

I test using: https://regex101.com

```
^\s*-:(?P<line>\d+):.+?: (?P<message>[^\r\n]+)(\r?\n[^\r\n]*\r?\n(?P<col>[^\^]*)\^)?
```
Gives correct matches for:

```
file:/c/Users/bmix/AppData/Roaming/Microsoft/Windows/Templates/xmlcatalog/catalog.xml:99: element uri: Catalog error : uri entry 'uri' broken ?: file://C:/Users/bmix/lib/uuid.xsl
    -:19: namespace error : Namespace prefix nml for role on date is not defined
        <dc:date nml:role="exported">2017-11-12T05-11+0100</dc:date>
```

Reply:
For safety? xmllint has no leading whitespace before the `-`

Reply:
Well, then you don't need it. I just copied the output from the issue report, but I guess that's indented and the regular output isn't. All good then.